pallets/settlement/src/lib.rs

```diff
@@ -1075,7 +959,9 @@ impl<T: Trait> Module<T> {
             instruction_details.status == InstructionStatus::Pending,
             Error::<T>::InstructionNotPending
         );
+        #[allow(unused_variables)]
         if let Some(valid_from) = instruction_details.valid_from {
+            #[cfg(not(feature = "runtime-benchmarks"))]
```

Comment:
Yeah, true but this will not give use anything productive for the benchmarks as reding the timestamp is free for benches as it lives in overlay and there is no way to set the timestamp in the `non_std` environment to validate the `ensure!`. So I think it is ok to have.

Reply:
Use this to set the time in wasm `<pallet_timestamp::Now<T>>::set(3000.into());`